### DISCUSSION TITLE
The tab indent size setting in the code editing window is not fixed.

### DISCUSSION THREAD

**Discussion by PeterSuh-Q3** [community/community#112644] [Repositories]
### Select Topic Area

Bug

### Body

The tab indent size setting in the code editing window is not fixed.

The tab indent of the edit window has been changed in the settings below.

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/managing-your-tab- size-rendering-preference

I wanted to use 4, so I set it to 4.

However, some scripts are well maintained at 4 as set,

In certain scripts it sometimes appears as 8.

https://github.com/PeterSuh-Q3/tinycore-redpill/blob/main/menu_m.sh

Even within the same repo, there is a problem where the number 4 or 8 appears differently for each script.

Could this be a web browser compatibility issue?

The web browser used is the Chromie family.

**Comment by MasterInQuestion** [comment]
Try similar local fix:
    https://github.com/orgs/community/discussions/106841#discussion-6197454
